### PR TITLE
Add in-tree docs for acc prov input file new parameters

### PR DESCRIPTION
**Pull request by pkharat**

### CODE REVIEW

**Comment by snaiksat** [comment]
hpp-optimization.md is missing

**Comment by pkharat** [comment]
> hpp-optimization.md is missing

Its there, the hpp related docs are put under newly created hpp section

**Comment by pkharat** [comment]
> > hpp-optimization.md is missing
> 
> Its there, the hpp related docs are put under newly created hpp section

https://github.com/noironetworks/acc-provision/blob/in-tree-docs/doc/features/hpp-optimization.md

File: doc/features/disable-hpp-rendering.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+* [Overview](#overview)
+* [Mechanism](#mechanism)
+* [Examples](#examples)
+
+## Overview
+
+Whenever a Network Policy is created in the cluster, a corresponding Host Protection Policy(HPP) is created in APIC. A HPP object is created in APIC with name `<policyTenantName>_np_<namespace>_<network-policy-name>` in `<Tenant> -> Policies -> Host Protection` tab.
+This Host Protection Policy functionality can be disabled, effectively disabling the Kubernetes Network Policy functionality through acc provision input file.
+
```
[the remark this answers is not in the document]
Comment (pkharat) [reply doc/features/disable-hpp-rendering.md:12]: Addressed

File: doc/features/README.md [hunk 2]
```diff
@@ -1042,7 +1042,9 @@ This section explains features / configuration options available in acc provisio
 - [Automatic ingress contract assignment for service type LoadBalancer](ingress-contract-assignment-for-lb-service.md)
 - [Exclude cluster nodes from snat redirect](node-snat-redirect-exclude.md)
 - [VMWare vSphere DVS ELAG](vsphere-dvs-elag.md)
-- [Host Protection Policies Optimization](hpp-optimization.md)
+- Host Protection Policy
+    - [Host Protection Policies Optimization](hpp-optimization.md)
+    - [Disable Host Protection Policy Rendering](disable-hpp-rendering.md)
```
[the remark this answers is not in the document]
Comment (pkharat) [reply doc/features/README.md:1048]: addressed

**Review by snaiksat — APPROVED** [review]

**Pull request merged**